### PR TITLE
adapt pad const

### PR DESCRIPTION
**Pull request by littletomatodonkey**
### PR types
New features

### PR changes
APIs

### Describe
adapt pad apis to contain the apis before.

### CODE REVIEW

File: python/paddle/nn/functional/common.py [hunk 1]
```diff
@@ -1273,6 +1274,9 @@ def pad(x, pad, mode='constant', value=0, data_format="NCHW", name=None):
 
     unsqueezed_dim = []
 
+    if mode == "constant" and isinstance(pad, list) and len(pad) == x_dim * 2:
```
Comment (willthefrog) [inline python/paddle/nn/functional/common.py:1279]: would be nice to add some comments about how this works

File: python/paddle/nn/functional/common.py [hunk 2]
```diff
@@ -14,6 +14,7 @@
 
 import warnings
 import paddle
+from paddle import fluid
```
Comment (willthefrog) [inline python/paddle/nn/functional/common.py]: why?

File: python/paddle/fluid/tests/unittests/test_pad3d_op.py [hunk 3]
```diff
@@ -251,7 +251,9 @@ def _get_numpy_out(self,
                        mode,
                        value=0,
                        data_format="NCDHW"):
-        if data_format == "NCDHW":
+        if mode == "constant" and len(pad) == len(input_data.shape) * 2:
```
Comment (willthefrog) [inline python/paddle/fluid/tests/unittests/test_pad3d_op.py:254]: better comment in code that the padding input is different

Comment (littletomatodonkey) [reply python/paddle/fluid/tests/unittests/test_pad3d_op.py:254]: 代码中已添加注释，谢谢~

File: python/paddle/nn/functional/common.py (hunk 2 above)
Comment (littletomatodonkey) [reply python/paddle/nn/functional/common.py]: 已删除~

File: python/paddle/nn/functional/common.py (hunk 1 above)
Comment (littletomatodonkey) [reply python/paddle/nn/functional/common.py:1279]: api注释中和代码中添加注释，多谢提醒

File: python/paddle/nn/functional/common.py [hunk 4]
```diff
@@ -1273,6 +1276,12 @@ def pad(x, pad, mode='constant', value=0, data_format="NCHW", name=None):
 
     unsqueezed_dim = []
 
+    if mode == "constant" and isinstance(pad, list) and len(pad) == x_dim * 2:
+        print(
```
Comment (willthefrog) [inline python/paddle/nn/functional/common.py]: please remove, too much noise in stdout.

**Review by willthefrog — CHANGES_REQUESTED** [review]

File: python/paddle/nn/functional/common.py (hunk 4 above)
Comment (littletomatodonkey) [reply python/paddle/nn/functional/common.py]: 已删除，多谢~

**Review by willthefrog — DISMISSED** [review]
LGTM

**Review by ceci3 — APPROVED** [review]

**Pull request merged**